tribe-ext-select-ticket-providers/src/Tribe/Settings_Helper.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+if ( class_exists( 'Tribe__Settings_Helper' ) ) {
```

Comment:
We're testing for the existence of `Tribe__Settings_Helper` (to prevent defining the same class twice), but in the actual definition the class is named `Tribe__Extension__Settings_Helper`.